core/simplerouter.go

```diff
@@ -35,7 +45,13 @@ func (router *SimpleRouter) Configure(conf PluginConfigReader) error {
 	router.Log = conf.GetLogScope()
 	router.Timeout = nil
 	router.streamID = conf.GetStreamID("Stream", GetStreamID(conf.GetID()))
-	router.modulators = conf.GetModulatorArray("Modulators", router.Log, ModulatorArray{})
+
+	// init router.modulators with filters only
+	filters := conf.GetFilterArray("Filters", router.Log, FilterArray{})
+	for _, filter := range filters {
+		filterModulator := NewFilterModulator(filter)
```

Comment:
You can spare yourself this code if you have a filter array instead of a modulator array